### PR TITLE
Irina

### PR DESCRIPTION
**Pull request by IrinaBuhaniuc**
i will need hel by catching error from django and putting it in rest for IncomingInvoiceFilter, everything is comented there.

### CODE REVIEW

File: petoshield_api/apps/policy/tests/conftest.py [hunk 1]
```diff
@@ -0,0 +1,270 @@
...
+def provider_user(roles):
+    provider = get_user_model().objects.create_user(email='provider@mail.com',
+                                                    password='password1A@',
+                                                    name='Provider User',
+                                                    role=roles[2])
+    return provider
+
+
+@pytest.fixture
+def service_provider(simple_user):
```
Comment (vitaliisili) [inline petoshield_api/apps/policy/tests/conftest.py:56]: service provider should get provider_user fixture

File: petoshield_api/apps/policy/tests/conftest.py [hunk 2]
```diff
@@ -0,0 +1,270 @@
...
+            species='cat',
+            breed=breed,
+            user=staff_user
+        ),
+    ]
+    return pets_list
+
+
+@pytest.fixture
+def service_provider_list(service_provider, simple_user):
```
Comment (vitaliisili) [inline petoshield_api/apps/policy/tests/conftest.py:126]: get fixture from user list

File: petoshield_api/apps/user/tests/test_endpoints.py [hunk 3]
```diff
@@ -226,7 +226,99 @@ def test_user_pagination_not_found(self, staff_user, users_list, api_client, pag
...
+    @pytest.mark.parametrize('user_name, length', [('Example Name1', 1), ('Example', 0), (' ', 5)])
+    def test_user_filter_by_name_exact_success(self, staff_user, users_list, user_name, api_client, length):
+        api_client.force_authenticate(staff_user)
+        response = api_client.get(f'{self.endpoint}?name={user_name}')
+        assert response.status_code == 200
+        assert len(json.loads(response.content)) == length
+        
+    # test filter by is_active
+    @pytest.mark.parametrize('is_active_, length', [('true', 5), ('TRUE', 5), (' ', 5), ('false', 0), ('FALSE', 0), ('T', 5)])
+    def test_user_filter_by_is_active_success(self, staff_user, users_list, is_active_, api_client, length):
```
Comment (vitaliisili) [inline petoshield_api/apps/user/tests/test_endpoints.py:317]: remove icontains

File: petoshield_api/apps/user/filters.py [hunk 4]
```diff
@@ -17,9 +17,8 @@ class UserFilter(filters.FilterSet):
     class Meta:
         model = User
         fields = {
-            'name': ['iexact', 'icontains'],
+            'name': ['exact', 'icontains'],
             'email': ['exact'],
             'created_at': ['exact', 'gt', 'lt'],
-            'is_active': ['icontains'],
-            'created_at': ['exact', 'gt', 'lt']
+            'is_active': ['icontains']
```
Comment (vitaliisili) [inline petoshield_api/apps/user/filters.py:23]: change icontains to exact

**Review by vitaliisili — CHANGES_REQUESTED** [review]

**Review by vitaliisili — CHANGES_REQUESTED** [review]
Please fix all sugestion that I write and sent to review again

**Review by timo9067 — APPROVED** [review]

**Review by JulienSchoener — APPROVED** [review]

**Review by vitaliisili — APPROVED** [review]

**Pull request merged**